sound/soc/sof/sof-audio.h

```diff
@@ -110,8 +110,7 @@ struct snd_sof_widget {
 	struct list_head list;	/* list in sdev widget list */
 	struct snd_sof_widget *pipe_widget;
 
-	/* extended data for UUID components */
-	struct sof_ipc_comp_ext comp_ext;
+	u8 uuid[SOF_UUID_SIZE];
```

Comment:
do we need to remove the definitions for sof_ipc_comp_ext then?

````c
/* extended data struct for UUID components */
struct sof_ipc_comp_ext {
	uint8_t uuid[SOF_UUID_SIZE];
}  __packed;
````

Reply:
yes, removed now.